### ISSUE TITLE
Add /var/lib/jellyfin to Mediaserver TKLBAM profile

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#1527]
As suggested by [Ole in the forums](https://www.turnkeylinux.org/forum/support/sat-20201017-1312/error-updating-mediaserver#comment-49032) it seems likely that we should include `/var/lib/jellyfin` in the backup profile for the Mediaserver appliance.

<s>At this point, this still needs to be confirmed.</s>

**Comment by JedMeister** [comment]
Workaround is to add `jellyfin` to the TKLBAM overrides (and exclude the same dirs as we did with Emby):
```
cat >> /etc/tklbam/overrides <EOF
/var/lib/jellyfin
-/var/lib/jellyfin/cache/temp
-/var/lib/jellyfin/transcoding-temp
```

**Comment by JedMeister** [comment]
Closed by https://github.com/turnkeylinux/tklbam-profiles/commit/560406d11737f73a8f07f7a9936ae64583f15e13

(Will be included in v16.1 release).
